core/templates/dev/head/pages/base.html

```diff
@@ -122,75 +120,68 @@
   </head>
 
   <body>
-    {% if iframed %}
-      {{ warnings_and_loader() }}
-    {% else %}
-      <div role="button" tabindex="0" ng-click="skipToMainContent()" class="oppia-skip-to-content protractor-test-skip-link">Skip to Main Content</div>
-      <promo-bar ng-if="promoBarIsEnabled" promo-message="promoBarMessage">
-      </promo-bar>
-      <div ng-if="isBackgroundMaskActive()" class="ng-cloak oppia-background-mask">
-      </div>
+    <span ng-if="isIframed" translate="{{ warnings_and_loader() }}"></span>
+    <div role="button" tabindex="0" ng-click="skipToMainContent()" class="oppia-skip-to-content protractor-test-skip-link">Skip to Main Content</div>
```

Comment:
My originally intention was to do without a container div with `ng-if="!firamed"` since warnings_and_loader() looks like a failure function. I am going to add the container div in now!